x-pack/plugins/code/server/init.ts

```diff
@@ -260,7 +257,7 @@ async function initCodeNode(server: Server, serverOptions: ServerOptions, log: L
   lspRoute(codeServerRouter, lspService, serverOptions);
   setupRoute(codeServerRouter);
 
-  server.expose('stop', () => {
+  server.events.on('stop', () => {
     indexScheduler.stop();
```

Comment:
nit: might be noop, but I asked @spacedragon to apply the `serverOptions.disableIndexScheduler` flag in here, he might forget. can we add a flag in here?